core/storage/exploration/gae_models_test.py

```diff
@@ -64,6 +64,21 @@ def test_get_exploration_count(self):
         self.assertEqual(saved_exploration.category, 'A Category')
         self.assertEqual(saved_exploration.objective, 'An Objective')
 
+    def test_reconstitute(self):
+        exploration = exp_domain.Exploration.create_default_exploration(
+            'id', title='A Title',
+            category='A Category', objective='An Objective')
+        exp_services.save_new_exploration('id', exploration)
+        exp_model = exp_models.ExplorationModel.get_by_id('id')
+        snapshot_dict = exp_model.compute_snapshot()
+        snapshot_dict['skill_tags'] = ['tag1', 'tag2']
+        snapshot_dict['default_skin'] = 'conversation_v1'
+        snapshot_dict['skin_customizations'] = {}
+        exp_model = exp_models.ExplorationModel.get_version('id', 1)
+        for field in ['skill_tags', 'default_skin', 'skin_customization']:
+            self.assertNotIn(field, exp_model._properties) # pylint: disable=protected-access
+            self.assertNotIn(field, exp_model._values) # pylint: disable=protected-access
```

Comment:
Would it be possible to verify the dict instead of the model directly? You can probably use `convert_to_valid_dict` for the testing.

Reply:
Done

Reply:
Not done, AFAICT.

Please don't write "Done" unless the code changes are done, otherwise reviewers are not going to be able to trust that you're representing the state of things correctly and this will lead to longer review times..

Reply:
My bad. @seanlip @vojtechjelinek I misunderstood this completely. Will push these changes.

Reply:
@vojtechjelinek I have updated this PR with new changes. Please check whether this is what you meant? Or are there any changes to be done.